test/test_fx.py

```diff
@@ -73,5 +78,65 @@ def forward(self, d):
 
         self.assertEqual(out, ref_out)
 
+    def test_disallow_override(self):
+        # Custom delegate to disallow in-place tensor operations
+        class NoMutableCallDelegate(DefaultDelegate):
+            def create_node(self, graph : Graph, kind : str, target : Union[str, Callable],
+                            args : Tuple[Any], kwargs : Dict[str, Any], name : Optional[str] = None) -> Node:
+                name = target if isinstance(target, str) else torch.typename(target)
+                if name[-1] == '_':
+                    raise RuntimeError('In-place operations are not supported')
+                return super().create_node(kind, target, args, kwargs, name)
```

Comment:
nit: it seems to miss `graph` argument, but still works because we throw an error first